tests/src/core/RelativePathsTest.cpp

```diff
@@ -40,8 +40,10 @@ private slots:
 
 		QString absPath = fi.absoluteFilePath();
 		QString relPath = "drums/kick01.ogg";
+		QString fuzPath = absPath.replace(relPath, "drums/.///kick01.ogg");
```

Comment:
Current one won't work because this line modifies `absPath`. This one will work:
```cpp
QString fuzPath = absPath;
fuzPath.replace(relPath, "drums/.///kick01.ogg");
```

Reply:
Ah yes, forgot Strings were mutable in Qt.  Thanks.